script.js

```diff
@@ -53,13 +44,12 @@ function reloadCards() {
 //   }
 // }
 
-function checkNumberOfTasks() {
-  var ideaCardArray = $('.idea-card');
-  if (ideaCardArray.length > 10) {
-    return console.log('too many!');
-  }
-  else return console.log('you good');
-}
+// function checkNumberOfTasks() {
```

Comment:
This is fine because you're not ready to merge this in, but when you take off the `wip` or `question` label and this is actually ready you should get rid of this.